### DISCUSSION TITLE
Windows XP

### DISCUSSION THREAD

**Discussion by darksilicon** [rubberduck-vba/Rubberduck#6235] [Q&A]
What is the dependency on .net 4.6? It seems like running Rubber Ducky on XP so that you could support the (Win98) VB 6.0 IDE environment would be better or am I missing something?

**Comment by retailcoder** [comment]
At one point it was about balancing use cases and developer preferences: at the end of the .NET Framework lifecycle, C# started evolving very quickly and while we wanted Rubberduck to run on older machines, at one point it didn't make sense to continue building for a version of Windows that has been out of support for over a decade. The newer language and framework features have added significant enhancements all around, and the devs wanted to be able to use them in the RD code base, so at one point support for XP had to be dropped. IIRC it should still work on Vista, but if/when v3 happens it'll be under .net (core), without Vista support.

**Reply by Vogel612** [reply]
another consideration here is that RD3 (for future discussion purposes) wants to make use of the new language features and library support, so it makes no sense to keep it targeting net46. So Rd3 (if it happens) will not be compatible to that extent.
